Pipfile.lock

```diff
@@ -0,0 +1,741 @@
+{
```

Comment:
Does this file get updated with each release? If so, does that happen manually or is it automated somewhere?